tests/unittests/OnnxImporterTest.cpp

```diff
@@ -2268,3 +2268,33 @@ TEST(onnx, importWhere) {
   EXPECT_EQ(WHR->getResult().dims()[1], 4);
   EXPECT_EQ(WHR->getResult().dims()[2], 4);
 }
+
+TEST(onnx, importLess) {
+  ExecutionEngine EE{};
+  auto &mod = EE.getModule();
+  Function *F = mod.createFunction("main");
+
+  std::string netFilename(GLOW_DATA_PATH
+                          "tests/models/onnxModels/Less.onnxtxt");
+
+  Placeholder *out = nullptr;
+  {
+    Tensor X(ElemKind::FloatTy, {1, 4, 1});
+    Tensor Y(ElemKind::FloatTy, {4, 1, 1});
+    X.zero();
+    Y.zero();
+
+    ONNXModelLoader onnxLD(netFilename, {"X", "Y"},
+                           {&X.getType(), &Y.getType()}, *F);
+    out = EXIT_ON_ERR(onnxLD.getOutputByName("Out"));
+  }
+
+  auto *save = getSaveNodeFromDest(out);
+
+  CmpLTNode *CMPLT = llvm::dyn_cast<CmpLTNode>(save->getInput().getNode());
+
+  ASSERT_TRUE(CMPLT);
+  EXPECT_EQ(CMPLT->getResult().dims()[0], 4);
```

Comment:
nit: `ASSERT_EQ(CMPLT->getResult().dims().size(), 3);` before here to prevent potential segfault for this and below.